core/services/fluxmonitor/flux_monitor_simulated_blockchain_test.go

```diff
@@ -343,6 +349,13 @@ func TestFluxMonitorAntiSpamLogic(t *testing.T) {
 	nextRoundBalance := initialBalance - fee
 	// Triggers a new round, since price deviation exceeds threshold
 	atomic.StoreInt64(&reportPrice, answer+1)
+	cltest.WaitForRuns(t, j, app.Store, 2)
+	cltest.WaitForEthTxAttemptCount(t, app.Store, 2)
+	txa = cltest.GetLastEthTxAttempt(t, app.Store)
+	cltest.WaitForTxInMempool(t, fa.backend, txa.Hash)
+
+	fa.backend.Commit()
```

Comment:
```suggestion
      cltest.WaitForRunsAndAttemptsCount(t, j, app.Store, 2, fa.backend)
```
I see these lines repeated in a lot of places. Do you think it would be worth extracting a helper to cut down on boilerplate?

Reply:
yeah good call :+1: